data/formatters/generic.yaml

```diff
@@ -151,6 +161,51 @@ short_source: 'LOG'
 source: 'Cron log'
 ---
 type: 'basic'
+data_type: 'text:entry'
+message: '{text}'
+short_message: '{text}'
+short_source: 'LOG'
+source: 'Text File'
+---
+type: 'conditional'
+data_type: 'vsftpd:log'
+message:
+- '{text}'
+short_message:
+- '{text}'
+short_source: 'LOG'
+source: 'vsftpd log'
+---
+type: 'conditional'
+data_type: 'xchat:log:line'
+message:
+- '[nickname: {nickname}]'
+- '{text}'
+short_message:
+- '[nickname: {nickname}]'
+- '{text}'
+short_source: 'LOG'
+source: 'XChat Log File'
+---
+type: 'conditional'
+data_type: 'xchat:scrollback:line'
+message:
+- 'Time elapsed: {elapsed_seconds} seconds'
```

Comment:
I don't see elapsed_seconds in the original formatter, is this wai?

Reply:
ack, good catch